docs/arch.gv

```diff
@@ -114,17 +119,14 @@ digraph {
                     <tr>
                         <td>aec070645fe5...</td>
                         <td><i>[blob some repomd.xml]</i></td>
-                        <td>{ContentType: application/xml}</td>
```

Comment:
The table will be messed up now because these rows now only have 2 cells, while if you look at earlier rows in the same table, they have 3 cells, plus the header colspan is 3.

The other rows had ``-`` to denote "empty" for metadata, but now that there is always no metadata here, could we just delete the "metadata" column from all rows?